tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -148,7 +148,7 @@ public void CallToActionInvokeButton()
         }
 
         _log.Information("User clicked on the create environment button. Navigating to Select environment page in Setup flow");
-        _navigationService.NavigateTo(KnownPageKeys.SetupFlow, "startCreationFlow");
+        _navigationService.NavigateTo(KnownPageKeys.SetupFlow, "startCreationFlow;EnvironmentsLandingPage");

```

Comment:
typo?  Does `NavigateTo` expect a semi-colon delimited string?

Reply:
For this one, the second parameter of `NavigateTo` is an object or null. Its up to the page that the navigation redirects to handle the object. (They can choose not to handle it as well)